bash_completions.go

```diff
@@ -132,7 +132,10 @@ __handle_reply()
         declare -F __custom_func >/dev/null && __custom_func
     fi
 
-    __ltrim_colon_completions "$cur"
+    # available in bash-completion >= 2, not always present on macOS
+    if command -v __ltrim_colon_completions >/dev/null; then
```

Comment:
Doesn't command only return true is there is a binary named `__ltrim_colon_completions`? This would fail on every system everywhere, wouldn't it? Do you mean `declare -F` ?